scripts/yapf_formatter.py

```diff
@@ -0,0 +1,31 @@
+from yapf.yapflib.yapf_api import FormatCode
+from github import Github
+import sys
+import re
+import base64
+
+ACCESS_TOKEN = []
+REPO_NAME = []
+WORKING_PATH = []
+ACCESS_TOKEN = sys.argv[1]
+REPO_NAME = sys.argv[2]
+WORKING_PATH = sys.argv[3]
+
+g = Github(ACCESS_TOKEN)
+repo = g.get_repo(REPO_NAME)
+contents = repo.get_contents(WORKING_PATH)
+
+pyscripts = []
+for content_file in contents:
+    result = re.findall(r'\.[^.\\/:*?"<>|\r\n]+$', content_file.name)
```

Comment:
I am not sure that is this regex expression cross-platform(windows and linux)?

Also could you please explain why you choose to use `findall()` other than `match()`?

Also, this sentence should be improved. You can make it a seperated function to find the postfix and return it or match `.py` just in this regex expression.

Reply:
It was copied from a blog for reference. For simplicity, I've altered the regex expression and changed findall() into search().